priv/catalogue/message/example03.ex

```diff
@@ -0,0 +1,27 @@
+defmodule SurfaceBulma.Catalogue.Message.Example03 do
+  @moduledoc """
+  A closable message component
+  """
+
+  use Surface.Catalogue.Example,
+    subject: SurfaceBulma.Message,
+    catalogue: SurfaceBulma.Catalogue,
+    height: "600px",
+    title: "Message component (close button override)"
+
+  alias SurfaceBulma.Message
+  alias SurfaceBulma.Icon
+
+  def render(assigns) do
+    ~F"""
+    <Message id="example03" closable>
+      <Message.Header>
+        This is the header
+      </Message.Header>
+      <Message.Header.CloseButton aria_label="override" />
```

Comment:
This gave me a compilation error, but moving the label as a prop to the `Message` and removing the line fixed it.